modules/network-observability-configuring-CPU-memory-limit.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-config-CPU-memory-limit{context}"]
+
+= Increasing Network Observability operator memory limits
+You can increase memory limits for the Network Observability operator by patching the CSV:
```

Comment:
Not sure if we want to call out that this is for the Controller Manager pod specifically or not - @jotak @mariomac @jpinsonneau wdyt?

Reply:
@nathan-weinberg - @xenolinux is going to incorporate this info into Troubleshooting.

Reply:
Removed from Installing, as Servesha has placed in the Troubleshooting area.